Nanoleaf-wpf/Api/Effects/CustomColorEffect.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+using Winleafs.Api;
+using Color = System.Drawing.Color;
+
+namespace Winleafs.Wpf.Api.Effects
+{
+    public class CustomColorEffect : ICustomEffect
+    {
+        private readonly INanoleafClient _nanoleafClient;
+        private readonly Color _color;
+
+        public static string Name => $"{CustomEffectsCollection.EffectNamePreface}Custom Color";
+
+        public CustomColorEffect(INanoleafClient nanoleafClient, Color color)
+        {
+            _nanoleafClient = nanoleafClient;
+            _color = color;
+        }
+
+        public Task Activate()
+        {
+            return _nanoleafClient.StateEndpoint.SetHueAndSaturationAsync(
+                Convert.ToInt32(_color.GetHue()), 
+                Convert.ToInt32(_color.GetSaturation() * 100));
+        }
+
+        public Task Deactivate()
+        {
+            return _nanoleafClient.StateEndpoint.SetStateAsync(false);
```

Comment:
What does this line do?

Reply:
Turn off the nanoleaf to deactivate the color effect

Reply:
That is incorrect. The deactivate method is meant to stop things like timers. You cannot just blindy turn off the lights here, the orchestrator will do that when returning to the scehdule (or it will apply the current active effect of the schedule). You should remove this line and keep the method empty.

Reply:
Perhaps we should add some explanation to the docs of the interface to explain the active and deactivate methods

Reply:
Changed to returning a completed task and agree on the docs, later PR though